lib/pg.cursor.js

```diff
@@ -0,0 +1,145 @@
+'use strict';
+
+const common = require('metarhia-common');
+const knex = require('knex');
```

Comment:
We will not use `knex`

Reply:
I didn't want to write SQL generation library myself so I used what was already available. I'm not sure it is feasible to write whole of that library by ourselves. Though if we don't need all of the features and will only use a small subset of SELECT I think I can write that relatively quickly.

Reply:
Also, what are the reasons for not using `knex` or any other SQL generation library?
My other concern with implementing my own SQL generator is that it must be heavily tested and checked as it's a crucial part of this Cursor and any bug in it (and therefore incorrectly generated SQL) may result in hard to resolve consequences.

Reply:
See `knex` dependencies: https://www.npmjs.com/package/knex
Adding such libs makes dependencies uncontrolled.
We may see also dependencies of dependencies and sooner or later all npm may be found in `node_modules`.

On the other hand we need no complete SQL syntax. I have simple implementation used by many people for a long time. But I don't like this implementation: https://github.com/tshemsedinov/node-mysql-utilities/blob/bdcef8cef8718260d611b341a9b74bb6941334ae/utilities.js#L48

We can implement just a subset of SQL needed for provider/cursor, including: `SELECT, WHERE, ORDER, UNION`